app/res/layout/query_prompt_layout.xml

```diff
@@ -20,7 +20,7 @@
 
     <ImageButton
         android:id="@+id/prompt_hint_button"
-        android:layout_width="0dp"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_weight="0.5"
```

Comment:
we should be able to remove `layout_weight` if we are able to set the `layout_width` to `wrap_content`